cpp/models/abm/parameters.h

```diff
@@ -308,12 +293,29 @@ struct HighViralLoadProtectionFactor {
     }
 };
 
+/**
+ * @brief Effectiveness of a Mask of a certain MaskType% against an Infection% for the mask wearer.
+ */
+struct InwardMaskProtection {
```

Comment:
Keep in mind that this might have the same problems as the defaults for the protection levels in issue #1068 .